### PR TITLE
Akairo framework

### PR DESCRIPTION
**Pull request by ha6000**

### CODE REVIEW

File: src/listeners/ready.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class Ready extends Listener {
+    constructor() {
+        super('ready', {
+            emitter: 'client',
+            event: 'ready'
+        });
+    }
+    exec() {
+        console.log('I\'m ready!');
```
Comment (Rami-Sabbagh) [inline src/listeners/ready.ts:11]: Aren't there any inline way to define the listener?

File: .gitignore [hunk 2]
```diff
@@ -12,4 +12,4 @@ node_modules
 dist
 
 # Config
-src/config.ts
+config.ts
```
Comment (Rami-Sabbagh) [inline .gitignore:15]: Why moving my config file...

**Review by Rami-Sabbagh — COMMENTED** [review]
I would like to check the new discord slash commands system, before adding in a bot commands system

File: .gitignore (hunk 2 above)
Comment (ha6000) [reply .gitignore:15]: think it was a mistake, I think I changed it back in later version

File: src/listeners/ready.ts (hunk 1 above)
Comment (ha6000) [reply src/listeners/ready.ts:11]: most likely

**Review by ha6000 — COMMENTED** [review]
why :raised_eyebrow: ?

**Comment by Rami-Sabbagh** [comment]
Hey @ha6000 very sorry, a PR was made to use slash commands
Thanks for your help a bunch!

**Pull request closed without merging**